enhancements/external-control-plane-topology.md

```diff
@@ -0,0 +1,82 @@
+---
+title: External Control Plane Topology
+authors:
+  - "@csrwng"
+reviewers:
+  - "@derekwaynecarr"
+  - "@ironcladlou"
+  - "@enxebre"
+  - "@sjenning"
```

Comment:
why does this lack the leads of control-plane components?

Reply:
Sorry @sttts no reason at all. I added folks from hypershift by default but yes it really should include folks from various teams across OpenShift. Will update.